app-android/src/main/java/com/ustadmobile/port/android/view/ContentEntryDetailOverviewFragment.kt

```diff
@@ -346,4 +376,438 @@ class ContentEntryDetailOverviewFragment: UstadDetailFragment<ContentEntryWithMo
+                        color = contentColorFor(
+                            colorResource(id = R.color.primaryColor))
+                    )
+                }
+            }
+        }
+
+        item {
+            Spacer(modifier = Modifier.height(10.dp))
+        }
+
+        if (uiState.contentEntryButtons?.showOpenButton == true){
+            item {
+                Button(
+                    onClick = onClickOpen,
+                    modifier = Modifier
+                        .fillMaxWidth(),
+                    colors = ButtonDefaults.buttonColors(
+                        backgroundColor = colorResource(id = R.color.primaryColor)
+                    )
+                ) {
+                    Text(stringResource(R.string.open).uppercase(),
+                        color = contentColorFor(
+                            colorResource(id = R.color.primaryColor))
+                    )
+                }
+            }
+        }
+
+        items(
+            uiState.activeContentJobItems.size
+        ){
+            ContentJobListItem(
+                uiState = uiState,
+                onClickContentJobItem = onClickContentJobItem,
+                index = it
+            )
+        }
+
+        item {
+            Spacer(modifier = Modifier.height(10.dp))
+        }
+
+        if (uiState.locallyAvailable) {
+            item {
+                LocallyAvailableRow()
+            }
+        }
+
+        item {
+            Spacer(modifier = Modifier.height(10.dp))
+        }
+
+        item {
+            Divider(thickness = 1.dp)
+        }
+
+        item {
+            Spacer(modifier = Modifier.height(10.dp))
+        }
+
+        item {
+            QuickActionBarsRow(
+                uiState = uiState,
+                onClickMarkComplete = onClickMarkComplete,
+                onClickDelete = onClickDelete,
+                onClickManageDownload = onClickManageDownload
+            )
+        }
+
+        item {
+            Spacer(modifier = Modifier.height(10.dp))
+        }
+
+        item {
+            Text(text = "Locally Available")
+        }
+
+        item {
+            Spacer(modifier = Modifier.height(10.dp))
+        }
+
+        item {
+            Divider(thickness = 1.dp)
+        }
+
+        item {
+            Spacer(modifier = Modifier.height(10.dp))
+        }
+
+        item {
+            Text(text = uiState.contentEntry?.description ?: "")
+        }
+
+        item {
+            Spacer(modifier = Modifier.height(10.dp))
+        }
+
+        item {
+            Divider(thickness = 1.dp)
+        }
+
+        item {
+            Spacer(modifier = Modifier.height(10.dp))
+        }
+
+        if (uiState.translationVisibile){
+            item {
+                Text(text = stringResource(id = R.string.also_available_in))
+            }
+
+            items(
+                uiState.availableTranslations.size
+            ){
+                ListItem(
```

Comment:
The available translations is a list... but we should show this as buttons in rows as we are doing using XML. I think this can be done using https://google.github.io/accompanist/flowlayout/ . We are only showing the name of the language, so one language per row is taking up too much space. 

e.g. we should have:

English, Persian, Arabic

Instead of:

English
Persian
Arabic

We can't just use a column because there might be a lot of translations, we need something that will make a new line when required.